src/Scripting/CSharpTest.Desktop/CsiTests.cs

```diff
@@ -47,7 +48,17 @@ public void CurrentWorkingDirectory_Change()
 new C()
 Environment.Exit(0)
 ";
-            var result = ProcessUtilities.Run(CsiPath, "", stdInput: stdInput);
+            ProcessResult result = null;
```

Comment:
We should also set Console.OutputEncoding to an explicit value. All these Console settings are likely inherited from parent to child process.